ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1276,6 +1277,16 @@ daemon_terminattr:
 
 You can either provide a list of IPs to target on-premise Cloudvision cluster or either use DNS name for your Cloudvision as a Service instance. If you have both on-prem and CVaaS defined, only on-prem is going to be configured.
 
+#### Custom Daemons
+
+```yaml
+daemons:
+  < daemon_name >:
+    exec: "< command to run as a daemon >"
```

Comment:
I would suggest to add `enabled: < true | false >` knob to render `no shutdown` or `shutdown` on the daemon.